newIDE/app/src/InstancesEditor/InstancePropertiesEditor/index.js

```diff
@@ -133,6 +129,28 @@ export default class InstancePropertiesEditor extends Component {
           schema={this.schema.concat(instanceSchema)}
           instances={instances}
         />
+        <Line alignItems="center">
+          Instance Variables
+          <IconButton
+            tooltip={'Edit instance variables'}
+            onClick={() => {
+              this.props.editInstanceVariables(instance);
+            }}
+          >
+            <PopOut />
+          </IconButton>
+        </Line>
+        <VariablesList
+          inheritedVariablesContainer={object ? object.getVariables() : []}
```

Comment:
pass null instead of an empty array.
Passing an empty array is dangerous because you're telling the component that you'll be passing something that is a `gdVariablesContainer`, but sometime you're passing an array - so that things can crash if you're trying to use a function like `has`.

Never give yourself to the temptation of passing an empty object or an empty array "because it seems to work" while it should be something else. 99% of the case, it will result in a bug.